Doc/library/shutil.rst

```diff
@@ -247,6 +252,9 @@ Directory and files operations
    as arguments. By default, :func:`shutil.copy2` is used, but any function
    that supports the same signature (like :func:`shutil.copy`) can be used.
 
+   .. versionadded:: 3.8
```

Comment:
I put 3.8 here as a placeholder, but should this be 3.7.x? (3.7.1 assuming it's accepted in time?)